src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,84 @@
 package core.basesyntax;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exeption.FailedToAddUser;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private final RegistrationService registrationServiceImpl = new RegistrationServiceImpl();
+
+    @Test
+    void register_sameLogin_notOk() {
+        User user1 = new User(1L, "loginUser", "password", 18);
```

Comment:
Add the creation of a standard user through a method before each test, and in the test set the necessary non-standard parameters.
Also, do not forget to clear storage after the test.

Reply:
Just use AfterEach and inside the Storage.people.clear() method